### DISCUSSION TITLE
I can't login to my dogechain wallet

### DISCUSSION THREAD

**Discussion by Brac2020** [dogecoin/dogecoin#3698] [Help needed]
I had a dogechain wallet but fews years I am not login to my account due to my personal problem. But last week I'm trying to login dogechain wallet just say it's shut down. How can I get back my coins?

**Comment by admiralehab** [comment]
> I had a dogechain wallet but fews years I am not login to my account due to my personal problem. But last week I'm trying to login dogechain wallet just say it's shut down. How can I get back my coins?

What wallet was it dogechain? Ifso they just published a wallet backup so you can access it again to recover the private key

**Reply by Brac2020** [reply]
I have private keys but doesn't work

**Reply by admiralehab** [reply]
Did you find a wallet where you can import a wallet based on a private key?

**Reply by Brac2020** [reply]
Trying to get it. But sill looking

**Reply by Brac2020** [reply]
Can you help me ?

**Reply by admiralehab** [reply]
I think so if you can wait for 5 minutes i am back at my pc and will have a look

**Reply by admiralehab** [reply]
You can import a private key into meta mask

**Reply by admiralehab** [reply]
Atleast the chrome extension

**Reply by admiralehab** [reply]
![image](https://github.com/user-attachments/assets/7ed19dac-3100-42fe-80bf-8f333f8842c9). ![image](https://github.com/user-attachments/assets/e2377462-f3f9-4af7-b3ea-4a84c3da2cdb)

**Reply by admiralehab** [reply]
if it is not working try this instead https://anieljoshua94.medium.com/steps-to-bridge-withdraw-your-doge-from-to-dogecoin-network-to-from-dogechain-98bd6058e541

**Reply by admiralehab** [reply]
You can use guarda is tried it it works

**Reply by Brac2020** [reply]
![Screenshot_20241121_151730_Guarda](https://github.com/user-attachments/assets/d59ae66d-126c-4250-b009-0d7ed38c0fed)
Want firstly free

**Reply by Brac2020** [reply]
They ask fees

**Reply by admiralehab** [reply]
The only way to import it then is using a guardia wallet then you just import it so no fees

**Reply by Brac2020** [reply]
![Uploading Screenshot_20241121_154739_Guarda.jpg…]()

**Reply by Brac2020** [reply]
Can you give me any video?

**Reply by admiralehab** [reply]
https://youtu.be/D7iD3kVnihc?si=5Zj1ZR_xtHVgD7nV

**Reply by admiralehab** [reply]
> Can you give me any video?

Did it work?
